test/JDBC/expected/cast_eliminate-vu-verify.out

```diff
@@ -0,0 +1,82 @@
+set babelfish_showplan_all on
+GO
+
+SELECT 1 AS [C1] FROM cast_eliminate  WHERE (CAST(ROID AS BIGINT) = 1)
```

Comment:
To make out test suite more solid, I'd suggest to have the following test cases as well:
1. A case where cast should not be removed, e.g., BIGINT -> INT
2. A case where the right side of `=` is actually a bigint, e.g., CAST(ROID AS BIGINT) = (a real bigint number)